extism/iroh-extism-host-functions/src/lib.rs

```diff
@@ -22,36 +24,82 @@ pub async fn default_iroh_extism_data_root() -> Result<PathBuf> {
     Ok(path.join(IROH_EXTISM_DATA_DIR))
 }
 
-pub async fn create_iroh(path: PathBuf) -> Result<IrohNode> {
-    let node = iroh::node::Node::persistent(path).await?.spawn().await?;
-    Ok(node)
+pub struct Iroh {
```

Comment:
We should probably avoid calling this `Iroh`, but that's just a minor nit.

Reply:
How about `ExtismIroh`? Maybe `IrohBlobs`?

Reply:
it's extism iroh struct, so seems fine to me, better than `Node` 😅

Reply:
I'm going to keep it `Iroh`, to me it's "all the iroh stuff".